### PR TITLE
Default forum translation

### PR DESCRIPTION
**Pull request by Ikerepc**
Here I suggest adding guide how to change default board translation

### CODE REVIEW

File: 1.8/translations/translations.md [hunk 1]
```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
 #### Finishing Touches
 
 Go to **User CP > Edit Options**. You should see the translation in the **Other Options** menu. Try out the new translation and make sure that everything works.
+
+### Default forum translation
+
+If you want your translation to be default on forum so when users and guests access it you can change it by going to **Admin CP > Configuration > Settings > General Configuration > Default Language**.
```
Comment (Ben-MyBB) [inline 1.8/translations/translations.md]: If you want your translation to be default on forum so when users and guests access it you can change it by going to

Should be imo:

If you want your new translation to be default for all new users and guests you can change it by going to

File: 1.8/translations/translations.md [hunk 2]
```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
...
+That will make all new users and guests to see board firstly on that translation.
+
+**Please note:** Changing this in Admin CP will not force new default translation to existing users on board. They need to change it manually in their profiles.
+
+#### Default translation for Admin CP
+
+You can make default Admin CP translation to be different than default board translation.
```
Comment (Ben-MyBB) [inline 1.8/translations/translations.md]: You can make default Admin CP translation to be different than default board translation.

Should be imo:

You can set the default Admin CP language to be different than default forums translation.

File: 1.8/translations/translations.md [hunk 3]
```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
 #### Finishing Touches
 
 Go to **User CP > Edit Options**. You should see the translation in the **Other Options** menu. Try out the new translation and make sure that everything works.
+
+### Default forum translation
+
+If you want your translation to be default on forum so when users and guests access it you can change it by going to **Admin CP > Configuration > Settings > General Configuration > Default Language**.
+That will make all new users and guests to see board firstly on that translation.
```
Comment (Ben-MyBB) [inline 1.8/translations/translations.md]: ** This would make this the default translation for all new users and guests.

**Comment by Ikerepc** [comment]
Yeah, thanks for updates...

File: 1.8/translations/translations.md [hunk 4]
```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
 #### Finishing Touches
 
 Go to **User CP > Edit Options**. You should see the translation in the **Other Options** menu. Try out the new translation and make sure that everything works.
+
+### Default forum translation
+
+If you want your new translation to be default for all new users and guests you can change it by going to **Admin CP > Configuration > Settings > General Configuration > Default Language**.
```
Comment (andrewjs18) [inline 1.8/translations/translations.md]: This should be:
If you want your new translation to be default for all new users and guests you can change it by going to **Admin CP > Configuration > Change Settings > General Configuration > Default Language**.

File: 1.8/translations/translations.md [hunk 5]
```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
...
+This would make this the default translation for all new users and guests.
+
+**Please note:** Changing this in Admin CP will not force new default translation to existing users on board. They need to change it manually in their profiles.
```
Comment (andrewjs18) [inline 1.8/translations/translations.md]: this should be:
**Please note:** Changing this in the Admin CP will not force a new default translation for existing users on the forum. They will need to change it manually in their profiles.

File: 1.8/translations/translations.md [hunk 6]
```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
...
+
+#### Default translation for Admin CP
+
+You can set the default Admin CP language to be different than default forums translation.
+To do that you should go to **Admin CP > Home > Preferences > Admin Control Panel Language**
+
+Note that you have options **Default** and languages installed on your board. Option Default will make default Admin CP translation same as board translation.
```
Comment (andrewjs18) [inline 1.8/translations/translations.md]: This little section should be:
You can set the default Admin CP language to be different than the default forums translation.
To do that you should go to **Admin CP > Home > Preferences > Admin Control Panel Language**

Note that you have the options **Default** and languages installed on your board. Option Default will make the default Admin CP translation the same as the forum translation.

**Comment by andrewjs18** [comment]
thanks

**Pull request merged**